scripts/webpack/webpack.config.perf.ts

```diff
@@ -11,7 +11,7 @@ const { paths } = config;
 const webpackConfig: webpack.Configuration = {
   name: 'client',
   target: 'web',
-  mode: 'development',
+  mode: !!argv.debug ? 'development' : 'production',
```

Comment:
double negation is not very explicit JS  lang feature that is usually also foreign to newbies. casting with constructor is recommended.

```suggestion
  mode: Boolean(argv.debug) ? 'development' : 'production',
```